functions/exercise2/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+   padding: 20px;
+   background-color: #bdbdbd;
+}
+
+h1 {
+    text-align: center;
+    color: #330033;
+}
+
+
```

Comment:
Why there are so many empty lines here? One empty line dividing each style definition is sufficient.